logic/multisignature.js

```diff
@@ -120,12 +120,9 @@ Multisignature.prototype.verify = function(transaction, sender, cb) {
 	) {
 		return setImmediate(
 			cb,
-			[
-				'Invalid multisignature min. Must be between',
-				MULTISIG_CONSTRAINTS.MIN.MINIMUM,
-				'and',
-				MULTISIG_CONSTRAINTS.MIN.MAXIMUM,
-			].join(' ')
+			`Invalid multisignature min. Must be between ${
+				MULTISIG_CONSTRAINTS.MIN.MINIMUM
+			} and ${MULTISIG_CONSTRAINTS.MIN.MAXIMUM}`
```

Comment:
You’re right. I think Prettier formatted it so it doesn’t exceed line character limit cause I initially wrote the string literal in one line. Will take a look tomorrow. Thanks for pointing it out.

Reply:
Not related to this PR but I think we should talk about prettier settings as in many PRs it is introducing this kind of changes that I think are not always good.

Reply:
@pablitovicente @lsilvs [here](https://repl.it/@DanielLimia/multilinestringliterals). It seems it returns only one line as it takes advantage of inline variables to avoid long strings.

Reply:
That's true. Prettier is quite smart after all :)

Reply:
Resolving :)